app/main.py

```
@@ -2,0 +1,19 @@
+from app.cafe import Cafe
+from app.errors import (
+    VaccineError,
+    NotWearingMaskError
+)
+
+
+def go_to_cafe(friends: list[dict], cafe: Cafe | str) -> None | str:
+    masks_to_buy = 0
+    for friend in friends:
+        try:
+            Cafe(cafe).visit_cafe(friend)
+        except VaccineError:
+            return "All friends should be vaccinated"
```

Code Style: Here you are returning a string when an exception is raised. Instead of returning a string, it would be better to raise a custom exception with a descriptive error message. This would make your code more robust and easier to debug.